update-asdf-and-dockerfile/action.yml

```diff
@@ -82,22 +82,27 @@ runs:
       shell: bash
       run: |
         test -n "${{ inputs.docker_image }}" && test -f Dockerfile && sed -i "s/^FROM ${{ inputs.docker_image }}:${{ env.CURRENT_VERSION }}/FROM ${{ inputs.docker_image }}:${{ env.LATEST_VERSION }}/" Dockerfile
+    # We need to actually install nodejs to see the bundled npm version of the new nodejs
     - name: Update latest versions to package.json engines if it exists
+      if: ${{ inputs.plugin == 'nodejs' }}
       shell: bash
       run: |
         test -f package.json && \
-        sudo apt-get update && \
-        sudo apt-get install jq && \
-        asdf install && \
+        echo "package.json engines before update:" && \
+        jq ".engines" package.json && \
+        asdf install nodejs ${{ env.LATEST_VERSION }} && \
+        asdf global nodejs ${{ env.LATEST_VERSION }} && \
         cat package.json | jq ".engines.node |= \"^$(node --version | tr -d v)\"" | jq ".engines.npm |= \"^$(npm --version)\"" > updated-package.json && \
-        mv updated-package.json package.json
+        mv updated-package.json package.json && \
+        echo "package.json engines after update:" && \
+        jq ".engines" package.json
 
     # This is using the version 4.2.3
     # Reference the repository with SHA for security
     # Source: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
     - uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04
       with:
-        add-paths: '.tool-versions,Dockerfile'
+        add-paths: '.tool-versions,Dockerfile,package.json'
```

Comment:
@katalaakkonen this line was still missing and the `create-pull-request` action needs list of all files that it will include the PR 👍.

I'll merge this now and retry